shared/teams/routes.js

```diff
@@ -1,8 +1,16 @@
 // @flow
 import TeamsContainer from './index'
 import {RouteDefNode} from '../route-tree'
+import NewTeamDialog from '../chat/conversation/new-team/container'
```

Comment:
It feels like `NewTeamDialog` should be moved out into a common area...

Reply:
yes, this likely should go in /team

Reply:
Done. Moved everything related.